Docker/startozwdaemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+shopt -s nocaseglob
+if [ ! -c "$USB_PATH" ]
+then
+	echo "USB Path \"$USB_PATH\" does not exist or is not a Character Device"
+	exit 128
+else
+	CMD="-s $USB_PATH"
+fi
+
+if [ ! -z "$MQTT_SERVER" ]
+then
+	CMD+=" --mqtt-server $MQTT_SERVER"
```

Comment:
This does not handle whitespace very well, would be easy to fix by collecting to an array instead, e.g.
```sh
CMD=()
...
CMD+=(--mqtt-server "$MQTT_SERVER")
CMD+=(--mqtt-tls)
/usr/local/bin/ozwdaemon "${CMD[@]}"

Reply:
1) it appear the QT Command Line Parser doesn't handle quotes (") in arguments (it passes the quote through)
2) using a array - I'm ok.